pandapower/plotting/plotly/traces.py

```diff
@@ -575,7 +576,7 @@ def _create_branch_trace(net, branches=None, use_branch_geodata=True, respect_se
         except:
             pass
     if len(no_go_branches) > 0:
-        no_go_branches_to_plot = net[branch_element].loc[no_go_branches]
+        no_go_branches_to_plot = net[branch_element].loc[list(no_go_branches)]
```

Comment:
no_go_branches is defined in line 470 with set operators. It is possible to define it with the numpy set functions instead, and this way casting to list is also not necessary

Reply:
but considering the comments about performance above, the code here is OK